src/libraries/System.Xml.ReaderWriter/ref/System.Xml.ReaderWriter.cs

```diff
@@ -1379,7 +1398,7 @@ public partial class ValidationEventArgs : System.EventArgs
         public string Message { get { throw null; } }
```

Comment:
double check `public System.Xml.Schema.XmlSchemaException Exception { get { throw null; } }` is returning non-null

Reply:
looks like it's ok as is